modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonClientExperimentalCodegen.java

```diff
@@ -48,6 +61,19 @@ public PythonClientExperimentalCodegen() {
                 .build();
     }
 
+    @Override
+    public void processOpts() {
+        super.processOpts();
+        modelTemplateFiles.put("python-experimental/model.mustache", ".py");
+        supportingFiles.add(new SupportingFile("python-experimental/model_utils.mustache", packagePath(), "model_utils.py"));
+        apiTemplateFiles.put("python-experimental/api.mustache", ".py");
+        supportingFiles.add(new SupportingFile("python-experimental/api_client.mustache", packagePath(), "api_client.py"));
```

Comment:
I think you will need to first remove the one created in PythonClientCodegen (parent class of PythonClientExperimentalCodegen)

Same for apiTemplateFiles and modelTemplateFiles as these files are generated twice. Please see my test result (partial) below:

```
[main] INFO  o.o.codegen.AbstractGenerator - writing file /Users/williamcheng/Code/openapi-generator/samples/client/petstore/python-experimental/petstore_api/api/another_fake_api.py
[main] INFO  o.o.codegen.AbstractGenerator - writing file /Users/williamcheng/Code/openapi-generator/samples/client/petstore/python-experimental/petstore_api/api/another_fake_api.py
[main] INFO  o.o.codegen.DefaultGenerator - File exists. Skipped overwriting /Users/williamcheng/Code/openapi-generator/samples/client/petstore/python-experimental/test/test_another_fake_api.py
[main] INFO  o.o.codegen.AbstractGenerator - writing file /Users/williamcheng/Code/openapi-generator/samples/client/petstore/python-experimental/docs/AnotherFakeApi.md
[main] INFO  o.o.codegen.AbstractGenerator - writing file /Users/williamcheng/Code/openapi-generator/samples/client/petstore/python-experimental/docs/AnotherFakeApi.md
[main] INFO  o.o.codegen.AbstractGenerator - writing file /Users/williamcheng/Code/openapi-generator/samples/client/petstore/python-experimental/petstore_api/api/fake_api.py
[main] INFO  o.o.codegen.AbstractGenerator - writing file /Users/williamcheng/Code/openapi-generator/samples/client/petstore/python-experimental/petstore_api/api/fake_api.py
[main] INFO  o.o.codegen.DefaultGenerator - File exists. Skipped overwriting /Users/williamcheng/Code/openapi-generator/samples/client/petstore/python-experimental/test/test_fake_api.py
[main] INFO  o.o.codegen.AbstractGenerator - writing file /Users/williamcheng/Code/openapi-generator/samples/client/petstore/python-experimental/docs/FakeApi.md
[main] INFO  o.o.codegen.AbstractGenerator - writing file /Users/williamcheng/Code/openapi-generator/samples/client/petstore/python-experimental/docs/FakeApi.md
[main] INFO  o.o.codegen.AbstractGenerator - writing file /Users/williamcheng/Code/openapi-generator/samples/client/petstore/python-experimental/petstore_api/api/fake_classname_tags_123_api.py
[main] INFO  o.o.codegen.AbstractGenerator - writing file /Users/williamcheng/Code/openapi-generator/samples/client/petstore/python-experimental/petstore_api/api/fake_classname_tags_123_api.py
```

Reply:
Thanks for letting me know that. I have fixed this.